src/datachain/sql/functions/array.py

```diff
@@ -51,8 +51,16 @@ class avg(GenericFunction):  # noqa: N801
     inherit_cache = True
 
 
+class group_concat(GenericFunction):  # noqa: N801
```

Comment:
This function is not implemented in ClickHouse, so we need custom function, see related Studio PR.